src/NuGet.Indexing/RegistrationAddresses.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace NuGet.Indexing
+{
+    public class RegistrationAddresses
```

Comment:
All of these should be `BaseAddresses`, not just `Addresses`. It's fairly confusing for the terminology to jump back and forth.